analyzer/java/com/engflow/bazel/invocation/analyzer/bazelprofile/BazelProfile.java

```diff
@@ -77,69 +77,77 @@ public static BazelProfile createFromPath(String path) throws IllegalArgumentExc
 
   public static BazelProfile createFromInputStream(InputStream inputStream)
       throws IllegalArgumentException {
-    JsonObject bazelProfile;
-    try {
-      bazelProfile = JsonParser.parseReader(new InputStreamReader(inputStream)).getAsJsonObject();
-    } catch (IllegalStateException e) {
-      throw new IllegalArgumentException("Could not parse Bazel profile.", e);
-    }
-    return new BazelProfile(bazelProfile);
+    return new BazelProfile(new JsonReader(new InputStreamReader(inputStream)));
```

Comment:
When constructing an `InputStreamReader`, you should *always* pass in a `Charset`, such as `StandardCharsets.UTF_8`.

Reply:
Done.